src/kafka/account-generator/AccountGenerator.ts

```diff
@@ -24,19 +24,19 @@ export class AccountGenerator {
    *
    * @param producer - Producer used to interact with Orchestrate
    */
-  constructor(brokers: string[]) {
-    this.producer = new Producer(brokers, {
-      logLevel: logLevel.NOTHING,
-      clientId: 'wallet-generator-sdk-producer'
+  constructor(
+    brokers: string[],
+    kafkaConfig?: Omit<KakfaJS.KafkaConfig, 'brokers'>,
+    producerConfig?: KakfaJS.ProducerConfig,
```

Comment:
isn't there any default `producerCfg` we can set in behalf of user to facilitate its usage?